packages/theme/src/theme/_forms.checkbox.scss

```
@@ -76,9 +76,8 @@ $tt-checkbox-hover-border-color: tint($lochmara, 15) !default;
 	&:focus {
 		outline: none;
 
-		&:before,
-		+ *:before {
-			@include checkbox-outline;
+		+ label {
```

It doesn't work, you should let the previous selectors
```suggestion
		&:before, + *:before {
```